.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
```

Comment:
About this -> `private harassment` ... harassment is horrible of course. But if someone does that in private, out of Symfony repositories, out of Symfony conferences, etc. what should we do?

Reply:
Private harassment can be reported as stated here https://github.com/symfony/symfony/pull/24896/files#diff-510d358f0b955588b8abd91113ae6858R36
If it's not reported not much can be done..

Reply:
@javiereguiluz regarding the "should" do .. I guess what you are asking if we should even try to look at such stuff? I think the question is always if such private harassment has some relation to Symfony. for example if someone sends a problematic twitter DM with a twitter account that they associate with Symfony (either through the twitter bio or by using this account to interact with the Symfony community) then imho we should deal with this. If its a twitter account that is entirely unrelated to the Symfony community then based on my current thinking it would not *necessary* fall within the scope of this CoC.

Reply:
OK. So we should explicit that the harassment must be related to Symfony somehow, not matter if it's public or private.

Reply:
From what I have learned so far this specific topic is one of the trickiest aspects of CoCs since harassers tend to create dedicated troll accounts to specifically circumvent such clauses if they focus just on "public" harassment.